### DISCUSSION TITLE
ArduinoOTA does not work with SinricPro. Does anyone know the solution?

### DISCUSSION THREAD

**Discussion by basersoft** [JAndrassy/ArduinoOTA#237] [Q&A]
ArduinoOTA does not work with SinricPro. Does anyone know the solution? I couldn't fix the problem after the last updates. I need help from anyone who knows. The example given on the Sinric support page does not work either.

**Comment by JAndrassy** [comment]
I guess you didn't use this ArduinoOTA library, but the ArduinoOTA library bundled with the core for your esp8266, esp32 or RP2040
